src/components/HypothesisTesting/PerformTest.js

```diff
@@ -10,7 +10,7 @@ import DataDisplay from "./DataDisplay.js";
 import SampleSizeAlphaInputs from "./SampleSizeAlphaInput.js";
 import { popShapeType } from "../../lib/types.js";
 
-export default function PerformTest({ shape, tails, mue0 }) {
+export default function PerformTest({ distType, shape, tails, mue0 }) {
```

Comment:
Don't forget to add `distType` to the `PropTypes` for the component! I'd use `distributionType` from `src/lib/types.js`.